src/bin/download_sysext.rs

```diff
@@ -304,6 +304,10 @@ struct Args {
     /// may be specified multiple times.
     #[argh(option, short = 'm')]
     image_match: Vec<String>,
+
+    /// report download progress
+    #[argh(option, short = 'v', default = "false")]
+    print_progress: bool,
```

Comment:
How about having `#[argh(switch, short = 'v')]` instead here, so we can simply turn it on and off by `--print-progress`?

With `option`, we have to pass either `--print-progress true` or `--print-progress false`, (not even 1 or 0, not even a simple --print-progress).

Reply:
Good idea, done